Sources/XcodesKit/Environment.swift

```diff
@@ -300,7 +300,7 @@ public struct Network {
 
     public var validateSession: () -> Promise<Void> = client.validateSession
 
-    public var login: (String, String) -> Promise<Void> = { client.login(accountName: $0, password: $1) }
+    public var login: (String, String) -> Promise<Void> = { client.srpLogin(accountName: $0, password: $1) }
```

Comment:
Is it worth gating the old login approach behind a flag? This is the approach that [fastlane has taken](https://github.com/fastlane/fastlane/pull/26415).

Reply:
I don't see Apple ever returning back to their old style login. Xcodes doesn't have to be as backwards compatible as fastlane does, so I'm ok taking it out.